tests/backend/test_nongaussian_gates.py

```diff
@@ -18,13 +18,49 @@
 
 import numpy as np
 
-KAPPAS = np.linspace(0, 2 * np.pi, 7)
+from scipy.linalg import expm
 
+KAPPAS = np.linspace(0, 2 * np.pi, 7)
+GAMMAS = np.linspace(0, 2 * np.pi, 7)
```

Comment:
:laughing:
No need to use `pi` here, and might needlessly confuse people . Could just do something like `np.linspace(0, 6, 7)

Reply:
You're right, I was thinking to be consistent but it doesn't make sense to think of gamma as an angle. Thanks man!

Reply:
I've changed it in the new commit.